File: src/test/java/com/shelterhub/utils/MedicalRecordUtils.java
package com.shelterhub.utils;

import com.shelterhub.domain.model.MedicalRecord;
import com.shelterhub.dto.request.MedicalRecordRequest;

import java.util.UUID;

public class MedicalRecordUtils {

    public static MedicalRecord buildMedicalRecord() {

        return MedicalRecord.builder()
                .animalId(UUID.randomUUID())
                .id(UUID.randomUUID())
                .build();
    }


    public static MedicalRecordRequest buildMedicalRecordRequest() {
        return MedicalRecordRequest.builder()
                .animalId(UUID.randomUUID())
                .id(UUID.randomUUID())
                .build();
    }
}

File: src/test/kotlin/com/shelterhub/utils/MedicalRecordUtils.kt
package com.shelterhub.utils

import br.com.shelterhubmanagementapi.domain.model.MedicalRecord
import br.com.shelterhubmanagementapi.dto.request.MedicalRecordRequest
import br.com.shelterhubmanagementapi.dto.response.MedicalRecordResponse
import java.util.UUID

object MedicalRecordUtils {
    fun buildMedicalRecord() = MedicalRecord(
            animalId = UUID.randomUUID(),
            id = UUID.randomUUID()
        )

    fun buildMedicalRecordResponse() = MedicalRecordResponse(
        id = UUID.randomUUID(),
        animalId = UUID.randomUUID()
    )

    fun buildMedicalRecordRequest() = MedicalRecordRequest(
        animalId = UUID.randomUUID(),
        id = UUID.randomUUID()
    )
}
